src/main/java/Task.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Task is the main object used to maintain the list of tasks from the user by Duke.
+ *
+ * This class provides multiple implementations of Task object.
+ */
+public class Task {
```

Comment:
Could not find Todo, Event and Deadline classes in pull request. Could explore the possibility of creating these classes which extend Task class itself, thus using inheritance and polymorphism to make code more versatile.